targets/hasura/migrations/1613478848457_add_audit/up.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Audit the following table using audit triggers
+--
+
+select audit.audit_table('auth.refresh_tokens');
+select audit.audit_table('auth.user_roles');
+select audit.audit_table('auth.users', 'true', 'true', '{password,secret_token}');
+select audit.audit_table('alert_notes');
+select audit.audit_table('alert_status');
+select audit.audit_table('alerts', 'false');
+select audit.audit_table('document_relations', 'false');
+select audit.audit_table('glossary');
+select audit.audit_table('kali_blocks');
+select audit.audit_table('package_version');
+select audit.audit_table('roles');
+select audit.audit_table('sources');
+
+--
+-- select audit.audit_table('documents');
+-- we use audit trigger directly to take advantage of WHEN clause
+-- to log actions only if data updated since document is updated daily by the ingester,
+--
+CREATE TRIGGER documents_audit_update_selective
+  AFTER UPDATE ON documents FOR EACH STATEMENT
+  WHEN ( (OLD.title, OLD.meta_description, OLD.slug, OLD.is_published, OLD.is_searchable)
+  IS DISTINCT FROM (NEW.title, NEW.meta_description, NEW.slug, NEW.is_published, NEW.is_searchable) )
+  EXECUTE PROCEDURE audit.if_modified_func();
```

Comment:
This trigger doesn't work, `OLD` and `NEW` are null for statement trigger (https://www.postgresql.org/docs/current/plpgsql-trigger.html).

```suggestion
CREATE TRIGGER documents_audit_update_selective
AFTER UPDATE ON documents FOR EACH ROW
WHEN (OLD.title IS DISTINCT FROM NEW.title
    OR OLD.meta_description IS DISTINCT FROM NEW.meta_description
    OR OLD.slug IS DISTINCT FROM NEW.slug
    OR OLD.is_published IS DISTINCT FROM NEW.is_published
    OR OLD.is_searchable IS DISTINCT FROM NEW.is_searchable)
EXECUTE PROCEDURE audit.if_modified_func();
```